miniupnpd/netfilter_nft/nftnlrdr_misc.c

```diff
@@ -82,7 +83,37 @@ static uint32_t rule_list_filter_validate = RULE_CACHE_INVALID;
 static uint32_t rule_list_redirect_validate = RULE_CACHE_INVALID;
 static uint32_t rule_list_peer_validate = RULE_CACHE_INVALID;
 
+
+int
+nft_mnl_connect(void)
+{
+	int result = 0;
+
+	mnl_sock = mnl_socket_open(NETLINK_NETFILTER);
+	if (mnl_sock == NULL) {
+		log_error("mnl_socket_open() FAILED: %m");
+		result = errno;
+	} else {
+		if (mnl_socket_bind(mnl_sock, 0, MNL_SOCKET_AUTOPID) < 0) {
+			log_error("mnl_socket_bind() FAILED: %m");
+			result = errno;
+		} else {
+			mnl_portid = mnl_socket_get_portid(mnl_sock);
+		}
+	}
+	return result;
+}
+
+void
+nft_mnl_disconnect(void)
+{
+	mnl_socket_close(mnl_sock);
+	mnl_sock = NULL;
+}
+
 #ifdef DEBUG
+
+#if 0
```

Comment:
maybe you should comment why this code is left dead ;)

Reply:
I just removed it, it's been dead for a while :)  I just tend to tread lightly while I'm still getting up to speed on someone else's code.